src/sinks/websocket/config.rs

```diff
@@ -28,12 +28,25 @@ pub struct WebSocketSinkConfig {
     #[configurable(derived)]
     pub encoding: EncodingConfig,
 
-    /// The interval, in seconds, between sending PINGs to the remote peer.
+    /// The interval, in seconds, between sending [Ping][ping]s to the remote peer.
+    ///
+    /// If this option is not configured, pings are not sent on an interval. This option must be configured to a value > 0
+    /// in order to enable pings.
+    ///
+    /// If the `ping_timeout` is not set, pings are still sent but there is no expectation of pong
+    /// response times.
+    ///
+    /// [ping]: https://www.rfc-editor.org/rfc/rfc6455#section-5.5.2
+    #[configurable(metadata(docs::type_unit = "seconds"))]
     pub ping_interval: Option<u64>,
 
-    /// The timeout, in seconds, while waiting for a PONG response from the remote peer.
+    /// The number of seconds to wait for a [Pong][pong] response from the remote peer.
+    ///
+    /// If a response is not received within this time, the connection is re-established.
     ///
-    /// If a response is not received in this time, the connection is reestablished.
+    /// [pong]: https://www.rfc-editor.org/rfc/rfc6455#section-5.5.3
+    // NOTE: this option is not relevant if the `ping_interval` is not configured.
```

Comment:
This seems more user facing than dev facing?

Reply:
There is a cue warning about it on the user side. this was just kind of an after thought. I can remove it if it is counter productive

Reply:
Ah, that makes sense - it's mostly hard to tell there was user side docs there. One day we can add `docs::warning = ""`